enum/unterbrechbarkeit/unterbrechbarkeit.go

```diff
@@ -0,0 +1,12 @@
+package unterbrechbarkeit
+
+// Stromverbrauchsart/Unterbrechbarkeit Marktlokation
+
+//go:generate stringer --type Unterbrechbarkeit
+//go:generate jsonenums --type Unterbrechbarkeit
+type Unterbrechbarkeit int
+
+const (
+	UV Unterbrechbarkeit = iota + 1
```

Comment:
I checked that it's consistent with https://github.com/Hochfrequenz/BO4E-dotnet/blob/6b79477cb6799d71caf3eb2de54d9a8449ee30fc/BO4E/ENUM/Unterbrechbarkeit.cs#L10